template/en/default/mfa/page_requires_mfa.html.tmpl

```diff
@@ -0,0 +1,11 @@
+[% PROCESS "global/field-descs.none.tmpl" %]
+
+[% PROCESS global/header.html.tmpl
+   title  = "This page requires 2FA",
+   style_urls = ['skins/standard/page.css']         
+%]
+
+<p>Because you are a member of [% mfa_group FILTER html %] you must have 2FA enabled.
```

Comment:
i agree with dylan that `mfa_group` needn't be an array because of group inheritance, but the message here should reflect the group the user is a direct member of.

ie. if there's a `mfa-required` group which `mozilla-employee` is a member of, and i'm a member of `mozilla-employee`, this message should say "because you are a member of mozilla-employee", not "because you are a member of mfa-required".